src/framework/ui/uitypes.h

```diff
@@ -31,11 +31,13 @@
 #include "view/iconcodes.h"
 
 namespace mu::ui {
-static std::string DARK_THEME_CODE("dark");
-static std::string LIGHT_THEME_CODE("light");
-static std::string HIGH_CONTRAST_THEME_CODE("high_contrast");
+using ThemeCode = std::string;
 
-inline std::vector<std::string> allStandardThemeCodes()
+static ThemeCode DARK_THEME_CODE("dark");
+static ThemeCode LIGHT_THEME_CODE("light");
+static ThemeCode HIGH_CONTRAST_THEME_CODE("high_contrast");
```

Comment:
TODO needs to add const, like 
```
static const ThemeCode DARK_THEME_CODE("dark");
```

Reply:
added:
https://github.com/musescore/MuseScore/pull/7873/commits/30a8e4ac8e83d28c61e3d24f0ae887b56d14afa0